tests/integration/mounts_sshfs.bats

```diff
@@ -12,7 +12,19 @@ function setup() {
 	DIR="$BATS_RUN_TMPDIR/fuse-sshfs"
 	mkdir -p "$DIR"
 
-	if ! $sshfs -o ro rootless@localhost: "$DIR"; then
+	local flags=""
+
+	# Use different mount flags for fuse-sshfs depending on the test case.
+	case $BATS_TEST_NUMBER in
+	1)
+		flags="ro"
+		;;
```

Comment:
This seems super brittle. If we add a test case, we need to make sure it is not the first, or if we add any other test that takes other flags, we need to make sure the test number is mapped correctly to the flags here.

This really seems super brittle to me. Isn't there any other alternative?

Reply:
Better now?

Reply:
Not really. I mean, all the same applies but with the test name instead of the number. I think a bash function that creates the mount with the flags received by param makes more sense, as I suggested before.

Reply:
I am not sure I can follow. Instead of using the bats standard parameterless `setup` function you want to calll a parameterized "setup" function manually in each test before the `runc` call with specific parameters e.g. like `update_config`?

Reply:
Like now?

Reply:
LGTM :)